src/daemon/daemon.cpp

```diff
@@ -2201,6 +2143,7 @@ error_string mp::Daemon::async_wait_for_ssh_and_start_mounts_for(const std::stri
                 fmt::format_to(errors, "Removing \"{}\": {}\n", target_path, e.what());
                 invalid_mounts.push_back(target_path);
             }
+            persist_instances();
```

Comment:
I take it this was missing from before and you added it as a drive by fix not necessarily tied to what this PR is achieving?

Reply:
Yes I noticed that the mount was lost if the daemon crashed while creating it. Very edge-case :)

Reply:
Cool